libraries/bot-applicationinsights/pom.xml

```diff
@@ -63,6 +63,11 @@
     <dependency>
       <groupId>com.microsoft.bot</groupId>
       <artifactId>bot-builder</artifactId>
+      <scope>test</scope>
```

Comment:
Are we sure to use this as `test`? This is weird

Reply:
took it from [qna](https://github.com/southworks/botbuilder-java/blob/feature/southworks/libraries/application-insights/telemetry-initializer-tests/libraries/bot-ai-qna/pom.xml#L72), and fixed some import issues for us. But I'm not 100% sure

Reply:
So probably this is because the botbuilder is only used in the `test` side, we will understand this after having all the solution migrated. For now, it's ok